src/main/java/org/teacon/slides/ProjectorRenderer.java

```diff
@@ -1,12 +1,29 @@
 package org.teacon.slides;
 
+import com.google.common.collect.Streams;
 import com.mojang.blaze3d.matrix.MatrixStack;
 import com.mojang.blaze3d.vertex.IVertexBuilder;
-import net.minecraft.client.renderer.IRenderTypeBuffer;
-import net.minecraft.client.renderer.Matrix4f;
-import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.*;
```

Comment:
Not an urgent matter, but no star import, thanks

Reply:
Why not